_docs/geyser/extensions.md

```diff
@@ -0,0 +1,116 @@
+
+```java
+@Subscribe
+public void onPostInitialize(GeyserPostInitializeEvent event) {
+    // example: show that your extension is loading.
+    this.logger().info("Loading example extension...");
+}
+```
+
+To build your extension, run gradle build task, and install the extension.
+
+### Creating commands with Geyser Extensions
+To create a command, you would need to use the "Commands" package in the Geyser API. Brief rundown:
+- [Command.java](https://github.com/GeyserMC/Geyser/blob/master/api/src/main/java/org/geysermc/geyser/api/command/Command.java)
+  This interface represents a command in Geyser - to make one, you can use the CommandBuilder. You can register it with the
+  [GeyserDefineCommandsEvent](https://github.com/GeyserMC/Geyser/blob/master/api/src/main/java/org/geysermc/geyser/api/event/lifecycle/GeyserDefineCommandsEvent.java)
+- [CommandExecutor.java](https://github.com/GeyserMC/Geyser/blob/master/api/src/main/java/org/geysermc/geyser/api/command/CommandExecutor.java)
+  This interface represents a command execute handler in Geyser, and extends the CommandSource interface.
+- [CommandSource.java](https://github.com/GeyserMC/Geyser/blob/master/api/src/main/java/org/geysermc/geyser/api/command/CommandSource.java)
+  This interface represents a command source in Geyser. It can be used to e.g. send messages to the source, check if the source has permission to execute a command, or get the locale.
+
+```java
+Command command = Command.builder(this) // "this" is the extension's main class
+                .name("ExampleCommand")
+                .bedrockOnly(true) // only allow bedrock players to execute this command
+                .source(CommandSource.class) // set the source of the command
+                .aliases(List.of("example", "ex")) // set aliases for the command
+                .description("An example command") // set the description of the command
+                .executableOnConsole(false) 
+                .suggestedOpOnly(false)
+                .permission("example.command") // set the permission for the command
+                .executor((source, cmd, args) -> {
+                    // this is the command executor - this is where you would put your code to execute the command.
+                    // source is the source that executed the command
+                    // cmd is the command that was executed
+                    // args are the arguments passed to the command
+                    source.sendMessage("Hello World");
+                })
+                .build();
+```
+
+To register the command, you would need to subscribe to the GeyserDefineCommandsEvent, and register the command there:
+```java
+@Subscribe
+public void onDefineCommands(GeyserDefineCommandsEvent event) {
+    event.registerCommand(command);
+}
+```
+If everything went right, you should be able to execute the command in-game by running "/extesionid [command]" - in our case, "/exampleid examplecommand".
```

Comment:
```suggestion
If everything went right, you should be able to execute the command in-game by running "/extesionid [command]" - in our case, "/<exampleid> examplecommand".
```

Reply:
Doing this would result in the "exampleid" text not being shown at all due to markdown formatting - also, i am setting "exampleid" here as that is the id set in the exemplary extension.yml here